pyzx/__init__.py

```diff
@@ -20,6 +20,7 @@
 from .circuit import Circuit, gates, id
 from .linalg import Mat2
 from .utils import settings, VertexType, EdgeType
+from .mapping import to_tensor
```

Comment:
I'm not sure `mapping` is a very descriptive name. Maybe just call it `quimb` or `tensor_network`?